assets/js/modules/adsense/datastore/clients.test.js

```diff
@@ -133,5 +133,70 @@ describe( 'modules/adsense clients', () => {
 				expect( console ).toHaveErrored();
 			} );
 		} );
+
+		describe( 'getAFClient', () => {
+			it( 'returns undefined if clients are not yet resolved', async () => {
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/adsense\/data\/clients/,
+					{ body: fixtures.clients, status: 200 }
+				);
+
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				const initialAFCClient = registry
+					.select( MODULES_ADSENSE )
+					.getAFCClient( accountID );
+
+				expect( initialAFCClient ).toEqual( undefined );
+
+				await subscribeUntil(
+					registry,
+					() =>
+						registry
+							.select( MODULES_ADSENSE )
+							.getAFCClient( accountID ) !== undefined
+				);
+			} );
+
+			it( 'returns undefined if there are no clients', async () => {
```

Comment:
See above, this should return/expect `null` instead.

Reply:
👍